ROMFS/px4fmu_common/init.d/airframes/4020_holybro_px4vision_v1_5

```diff
@@ -0,0 +1,130 @@
+#!/bin/sh
+#
+# @name PX4 Vision Dev Kit v1.5
+#
+# @type Quadrotor x
+# @class Copter
+#
+# @board px4_fmu-v2 exclude
+# @board bitcraze_crazyflie exclude
+#
+
+. ${R}etc/init.d/rc.mc_defaults
+
+# Commander Parameters
+param set-default COM_OBS_AVOID 1
```

Comment:
Just curious, does this still work with the pixhawk 6c?

Reply:
We tested it in the factory, do you see an issue here? @Jaeyoung-Lim